.vscodeignore

```diff
@@ -14,7 +14,7 @@
 .vscode-test/**
 coverage/**
 out/**
-server/**
+msbuild/**
```

Comment:
Does this mean we will be packaging the ServerDownload.csproj into the extension?

Reply:
Never mind, I see now that it is nested.